app/controllers/api/automate_controller.rb

```diff
@@ -1,15 +1,14 @@
 module Api
   class AutomateController < BaseController
     def show
-      ae_browser = MiqAeBrowser.new(@auth_user_obj)
-      begin
-        resources = ae_browser.search(@req.c_suffix, ae_search_options)
-      rescue => err
-        raise BadRequestError, err.to_s
+      if browser_requested?
+        resources = collection_search
+        attributes = params['attributes'] ? %w(fqname) | params['attributes'].to_s.split(',') : nil
+        resources = resources.collect { |resource| resource.slice(*attributes) } if attributes.present?
+        render_resource :automate, :name => "automate", :subcount => resources.count, :resources => resources
+      else
+        super
       end
-      attributes = params['attributes'] ? %w(fqname) | params['attributes'].to_s.split(',') : nil
-      resources = resources.collect { |resource| resource.slice(*attributes) } if attributes.present?
-      render_resource :automate, :name => "automate", :subcount => resources.count, :resources => resources
     end
 
     def git_refresh_resource(type, id = nil, data = nil)
```

Comment:
I guess this was already here, but I don't like the action being named git_refresh...the fact that it's git under the covers should be obscured from the caller.  A more appropriate name might be `refresh` or `refresh_from_source`.  That way you can any number of backings, and they all refresh the same.  @abellotti